### PR TITLE
PERF-2182 Cleanup workloads

### PR DESCRIPTION
**Pull request by jimoleary**
fix clients, durations and transactionLfetimeLimitSeconds.

### CODE REVIEW

File: src/workloads/scale/LLTInsert.yml [hunk 1]
```diff
@@ -86,11 +86,11 @@ Actors:
           OperationName: RunCommand
           OperationCommand:
             setParameter: 1
-            transactionLifetimeLimitSeconds: 3600
+            transactionLifetimeLimitSeconds: 10800
 
 - Name: InitialLoad
   Type: Loader
-  ClientName: Loader
+  ClientName: Default
```
Comment (dalyd) [inline src/workloads/scale/LLTInsert.yml:93]: Default sounds like a boring name. Is there a reason to it?

File: src/workloads/scale/LLTInsert.yml [hunk 2]
```diff
@@ -86,11 +86,11 @@ Actors:
           OperationName: RunCommand
           OperationCommand:
             setParameter: 1
-            transactionLifetimeLimitSeconds: 3600
+            transactionLifetimeLimitSeconds: 10800
```
Comment (dalyd) [inline src/workloads/scale/LLTInsert.yml:89]: Could you add a trailing comment "# 3 hours" to each of these?

**Review by dalyd — APPROVED** [review]
Nit and a question

**Review by apodelko — APPROVED** [review]

File: src/workloads/scale/LLTInsert.yml (hunk 1 above)
Comment (jimoleary) [reply src/workloads/scale/LLTInsert.yml:93]: no, I'll remove them

File: src/workloads/scale/LLTInsert.yml (hunk 2 above)
Comment (jimoleary) [reply src/workloads/scale/LLTInsert.yml:89]: sure (and actually with an eye to the future, I will make it slightly longer).

**Comment by jimoleary** [comment]
Enough change to warrant a [patch](https://spruce.mongodb.com/version/608c37830305b933c21b8f14/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC)

**Review by dalyd — APPROVED** [review]

**Pull request merged**